proto/dymension/eibc/demand_order.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package dymensionxyz.dymension.eibc;
+
+import "gogoproto/gogo.proto";
+import "dymension/common/common.proto";
+
+option go_package = "github.com/dymensionxyz/dymension/v3/x/eibc/types";
+
+message DemandOrder {
+    // id is a hash of the form generated by GetRollappPacketKey, 
+    // e.g rollappid/status/packetProofHeight/packetDestinationChannel-PacketSequence which gurantees uniqueness
+    string id = 1;
+    // tracking_packet_key is the key of the packet that is being tracked. 
+    // This key can change depends on the packet status.
+    string tracking_packet_key = 2;
+    string price = 3;
+    string fee = 4;
```

Comment:
It's better to use the cosmos type like Coins, for this data
as it easier to parse and validate later on